extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -107,45 +100,28 @@ def package_url(node: 'Node') -> Optional[PackageURL]:
 
     def create_component(node: 'Node') -> Component:
         purl = package_url(node)
-        if cyclonedx_major_version_is_4():
-            component = Component(
-                type=package_type_to_component_type(node.conanfile.package_type),  # noqa
-                name=node.name or f'UNKNOWN.{id(node)}',
-                author=node.conanfile.author if node.conanfile.author else "Conan",
-                version=node.conanfile.version,
-                licenses=licenses(node.conanfile.license),
-                bom_ref=purl.to_string() if purl else None,
-                purl=purl,
-                description=node.conanfile.description
-            )
-        else:
-            component = Component(
-                component_type=package_type_to_component_type(node.conanfile.package_type),
-                name=node.name or f'UNKNOWN.{id(node)}',
-                author=node.conanfile.author,
-                version=node.conanfile.version,
-                licenses=licenses(node.conanfile.license),
-                bom_ref=purl.to_string() if purl else None,
-                purl=purl,
-                description=node.conanfile.description
-            )
-        if node.conanfile.homepage and cyclonedx_major_version_is_4():  # bug in cyclonedx 3 enforces hashes
+        component = Component(
+            type=package_type_to_component_type(node.conanfile.package_type),  # noqa
```

Comment:
why uis this `# noqa` needed?

Reply:
sorry, they were required when parts of the code were not tested in the CI, I dropped it